src/components/molecules/Footer/index.jsx

```diff
@@ -62,8 +62,12 @@ class Footer extends React.Component{
                 </MDBContainer>
                 <div className="footer-copyright text-center py-3">
                     <MDBContainer fluid>
-                        &copy; {new Date().getFullYear()} Copyright: 
-                        <a href="https://www.aichner-christian.com" target="_blank" rel="noopener noreferrer"> Werbeagentur Christian Aichner </a>
+                        &copy; 2018 - {new Date().getFullYear()} Copyright: Pharmaziegasse<sup>®</sup>
+                        <br />
+                        <p className="text-center my-2">
+                            Made with <i className="fas fa-heart pulse red-text" aria-hidden="true"></i> by
+                            <a target="_blank" rel="noopener noreferrer" href="https://www.aichner-christian.com?refer=charm">Werbeagentur Christian Aichner</a>
```

Comment:
NOTE: We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult.

Reply:
I will remember that for future commits